### DISCUSSION TITLE
Different heat set points for different circuit groups

### DISCUSSION THREAD

**Discussion by cmcollander** [tagyoureit/nodejs-poolController#897] [Nixie]
I have a nixie shared body system with a pool and a spa with a single VS pump, heater (fireman switch and 10k therms), and filter. I have set up simple circuit groups for the pool and the spa and can switch between them. Spa turns on the heat, and pool turns off (I set the heat set points in the main window).
Now, I want an additional circuit group for the pool with the heater enabled. Is there any way to do this?

**Comment by rstrouse** [comment]
Select the heater definition to be Pool/Spa.  It will then let you set a set point for the pool.

**Reply by cmcollander** [reply]
It is already set for pool/spa, and I can set different temps for each, but I was wondering if I can set this in a circuit group specifically. For example, have different modes for different temps. But I figure that just isn't possible in the code yet.

**Reply by rstrouse** [reply]
Only schedules can change the setpoints and turn the heater on and only when the schedule is for a body circuit.  This is by design because doing this from a group for instance has way too many undefined states.  However, if the pool circuit is on and the heater mode is set to heater the heater will automatically come on unless the schedule overrides it.
